debian/gatekeeper.cron.daily

```diff
@@ -3,6 +3,6 @@
 /usr/bin/find /var/log/gatekeeper \
   -maxdepth 1                     \
   -type f                         \
-  -name gatekeeper_*.log          \
-  -mtime +1                       \
+  -name gatekeeper_\*.log         \
+  -mtime 1                        \
```

Comment:
Shouldn't we drop the space after `-mtime 1` like you did after `.log` above?

Reply:
The spacing changes are meant to account for the spacing of the backslash characters that escape the newlines. In the `.log` line a space was removed because a backslash was inserted before the asterisk; in the `mtime` line, a space was inserted because the `+` character was removed. This is just so that the newline escapes are aligned correctly.